src/AlgoliaSearchBundle.php

```diff
@@ -15,7 +14,9 @@ public function boot()
     {
         parent::boot();
 
-        Version::addSuffixUserAgentSegment('Symfony Search Bundle', self::VERSION);
-        Version::addSuffixUserAgentSegment('Symfony', SfKernel::VERSION);
+        if (class_exists('Algolia\AlgoliaSearch\Support\Config')) {
```

Comment:
I'm not sure what the PHP requirement of this package is (there's none in `composer.json` which is bad enough), but starting with 5.5.0 there's the `::class` constant that you should use instead of magic strings. This allows IDEs to find code like this when looking for usages of a class.

Reply:
I didn't specify it because I'm maintaining the same as Symfony 3.4, which requires `^5.5.9`. I will update the PR. For the client, I'm maintaing 5.3 so I sort of lost the habit of `::class` ^^